apps/i18n/dance/zh_tw.json

```diff
@@ -10,5 +10,5 @@
   "danceFeedbackNeedDifferentDance": "您需要使用不同的舞蹈。",
   "danceFeedbackNeedEveryTwoMeasures": "確認您正在使用「每 2 個小節」程式積木",
   "danceFeedbackNeedMakeANewDancer": "使用「製作新內容」程式積木來建立第二個舞者。",
-  "measure": "小節:"
+  "measure": "小節："
```

Comment:
@Hamms There is a space (I haven't checked if this is an issue).  It doesn't seem as if it will cause any problems.